gammapy/utils/regions.py

```diff
@@ -99,3 +99,23 @@ def make_pixel_region(region, wcs=None):
         return region
     else:
         raise TypeError("Invalid type: {!r}".format(region))
+
+
+class SphericalCircleSkyRegion(CircleSkyRegion):
```

Comment:
I guess this class is to support the case of the circular region, where no WCS is needed, with a uniform API. I think it's a good idea. So +1 to keep, but I agree the better place would be the `regions` package...